src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -0,0 +1,193 @@
+  lineItemsMappingJSON,
+} = require('../../../../v0/sources/shopify/config');
+const {
+  createPropertiesForEcomEventFromWebhook,
+  getProductsFromLineItems,
+} = require('./serverSideUtlis');
+
+const NO_OPERATION_SUCCESS = {
+  outputToSource: {
+    body: Buffer.from('OK').toString('base64'),
+    contentType: 'text/plain',
+  },
+  statusCode: 200,
+};
+
+const identifyPayloadBuilder = (event) => {
+  const message = new Message(INTEGERATION);
+  message.setEventType(EventType.IDENTIFY);
+  message.setPropertiesV2(event, MAPPING_CATEGORIES[EventType.IDENTIFY]);
+  if (event.updated_at) {
+    // converting shopify updated_at timestamp to rudder timestamp format
+    message.setTimestamp(new Date(event.updated_at).toISOString());
+  }
+  return message;
+};
+
+const ecomPayloadBuilder = (event, shopifyTopic) => {
+  const message = new Message(INTEGERATION);
+  message.setEventType(EventType.TRACK);
+  message.setEventName(RUDDER_ECOM_MAP[shopifyTopic]);
+
+  const properties = createPropertiesForEcomEventFromWebhook(event);
+  message.properties = removeUndefinedAndNullValues(properties);
+  // Map Customer details if present
+  const customerDetails = get(event, 'customer');
+  if (customerDetails) {
+    message.setPropertiesV2(customerDetails, MAPPING_CATEGORIES[EventType.IDENTIFY]);
+  }
+  if (event.updated_at) {
+    message.setTimestamp(new Date(event.updated_at).toISOString());
+  }
+  if (event.customer) {
+    message.setPropertiesV2(event.customer, MAPPING_CATEGORIES[EventType.IDENTIFY]);
+  }
+  if (event.shipping_address) {
+    message.setProperty('traits.shippingAddress', event.shipping_address);
+  }
+  if (event.billing_address) {
+    message.setProperty('traits.billingAddress', event.billing_address);
+  }
+  if (!message.userId && event.user_id) {
+    message.setProperty('userId', event.user_id);
+  }
+  return message;
+};
+
+const trackPayloadBuilder = (event, shopifyTopic) => {
+  const message = new Message(INTEGERATION);
+  message.setEventType(EventType.TRACK);
+  message.setEventName(SHOPIFY_TRACK_MAP[shopifyTopic]);
+  // eslint-disable-next-line camelcase
+  const { line_items: lineItems } = event;
+  const productsList = getProductsFromLineItems(lineItems, lineItemsMappingJSON);
+  message.setProperty('properties.products', productsList);
+  return message;
+};
+
+const processEvent = async (inputEvent, metricMetadata) => {
+  let message;
+  const event = lodash.cloneDeep(inputEvent);
+  let redisData;
+  const shopifyTopic = getShopifyTopic(event);
+  delete event.query_parameters;
+  switch (shopifyTopic) {
+    case IDENTIFY_TOPICS.CUSTOMERS_CREATE:
+    case IDENTIFY_TOPICS.CUSTOMERS_UPDATE:
+      message = identifyPayloadBuilder(event);
+      break;
+    case ECOM_TOPICS.ORDERS_CREATE:
+    case ECOM_TOPICS.ORDERS_UPDATE:
+    case ECOM_TOPICS.CHECKOUTS_CREATE:
+    case ECOM_TOPICS.CHECKOUTS_UPDATE:
+      message = ecomPayloadBuilder(event, shopifyTopic);
+      break;
+    case 'carts_update':
```

Comment:
Is this event name still valid in pixel based flow ?

Reply:
The event name is valid but the logic related to redis is not needed with the cart update event. 
Removed it in a followup commit

Reply:
refer `carts_update` from ECOM_TOPICS enum. doesn't this fall under ecomm spec ?

Reply:
Done.